src/test/java/Homework18.java

```diff
@@ -0,0 +1,13 @@
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest {
+    @Test
+    public void playSong() throws InterruptedException{
+
+        navigateToPage();
+        provideEmail("anastasia.dovhal@testpro.io");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        clickPlayButton();
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**

For example 

![18](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/a7bd1b6a-cd0b-4d03-bdfa-124004958653)